### PR TITLE
Add Azure Container Registry support

### PR DESCRIPTION
**Pull request by karolz-ms**
Includes
- updates to README.md
- some test streamlining
- updated testify/assert test dependency to latest release

Hope this helps ☺️

### CODE REVIEW

**Comment by karolz-ms** [comment]
@stevesloka @oliverspryn @singram please let me know if there is anything I can do to help getting this PR approved. The ultimate goal is to have Azure Container Registry support in minikube.

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # MAINTAINER: Steve Sloka <steve@stevesloka.com>
 # If you update this image please bump the tag value before pushing.
 
-TAG = 1.9
+TAG = 1.10
```
Comment (stevesloka) [inline Makefile]: Please don't change versions in this, that is a separate process.

Comment (karolz-ms) [reply Makefile]: Fixed!

**Comment by karolz-ms** [comment]
Thank you for quick reply @stevesloka , much appreciated.

This will be tested and demoed by multiple people here at Microsoft and I am being paid to make it work. So if there are any problems with Azure, send them my way.

Thanks again!

**Comment by oliverspryn** [comment]
Thank you for your contribution @karolz-ms. We appreciate your support.

**Pull request merged**